.github/workflows/release.yml

```diff
@@ -146,3 +146,62 @@ jobs:
         env:
           TWINE_USERNAME: __token__
           TWINE_PASSWORD: ${{ secrets.PYPI_SECRET_TOKEN }}
+
+  setup-ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Building Unix dependencies
+        run: sudo apt-get install -y build-essential lcov gcovr xdg-utils
```

Comment:
Why can't you call the `actions/setup/action.yaml` as you did before?

Reply:
I can and I will. I just missed this.